.changeset/config.json

```diff
@@ -5,7 +5,7 @@
   "fixed": [],
   "linked": [],
   "access": "public",
-  "baseBranch": "next",
+  "baseBranch": "master",
```

Comment:
why not `next`?

Reply:
Because it was this way before and I only changed it initially to see if it will help pass the CI build. However, it didn't and since I do not fully understand what impact does this have, I decided to not touch it.